commit 42a8ce1f587f10aa896ece3edbb7d31b627447a0
Author: Stefan Reimer <stefan@zero-downtime.net>
Date:   Fri Jul 24 15:05:14 2020 +0100

    Actually use nodeSelector for the deamonset, to allow selecting onle workers having the EFS mounted

diff --git a/helm/templates/daemonset.yaml b/helm/templates/daemonset.yaml
index 7fcfc1e..bfe3496 100644
--- a/helm/templates/daemonset.yaml
+++ b/helm/templates/daemonset.yaml
@@ -22,12 +22,15 @@ spec:
     spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
+        {{- with .Values.nodeSelector }}
+        {{- . | toYaml | nindent 8 }}
+        {{- end }}
       hostNetwork: true
       priorityClassName: system-node-critical
       tolerations:
         - operator: Exists
         {{- with .Values.node.tolerations }}
-{{ toYaml . | indent 8 }}
+        {{- . | toYaml | nindent 8 }}
         {{- end }}
       containers:
         - name: efs-plugin
